extensions/eclipse-che-theia-file-sync-tracker/src/browser/status-bar-contribution.ts

```diff
@@ -0,0 +1,115 @@
+/********************************************************************************
+ * Copyright (C) 2020 Red Hat, Inc. and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { injectable, inject } from 'inversify';
+import { FrontendApplicationContribution } from '@theia/core/lib/browser';
+import { MessageService } from '@theia/core/lib/common';
+import { StatusBar, StatusBarAlignment, StatusBarEntry } from '@theia/core/lib/browser/status-bar/status-bar';
+import { CheApiService } from '@eclipse-che/theia-plugin-ext/lib/common/che-protocol';
+import URI from '@theia/core/lib/common/uri';
+import ReconnectingWebSocket from 'reconnecting-websocket';
+import { DisposableCollection, Disposable } from '@theia/core';
+
+@injectable()
+export class StatusBarFrontendContribution implements FrontendApplicationContribution {
```

Comment:
I would say class name and file name should be more specific according to its goal, but up to you.